demo/tsconfig.json

```diff
@@ -23,6 +23,7 @@
         "baseUrl": ".",
         "paths": {
             "*": [
+                "./node_modules/tns-core-modules/*",
```

Comment:
That's true. But is the recommendation for all NativeScript users that want to use AoT to prefix their modules (in both apps and plugins)? I don't see that happening in the [Groceries sample](https://github.com/NativeScript/sample-Groceries/blob/master/app/login/login.component.ts) for instance so this confuses me.

Is there a way to reproduce the problem with my suggested changes?